Rewrite of the Conda part of our install instructions; closes #277, closes #287, closes #288, closes #292 (#293)
coderefinery/installation: content/conda.md

available on all operating systems, and provides a good basis for reproducible
environments. However, it does not provide a graphical user interface during
installation.

But Miniforge is only one option. Unfortunately there are many options and a
lot of jargon.  Here is a crash course:

- **Conda** is a package manager: it allows distributing and
  installing packages, and is designed for complex scientific
  code.
- **Mamba** is a re-implementation of Conda to be much faster with
  resolving dependencies and installing things.
- An **environment** is a self-contained collections of packages
  which can be installed separately from others.  They are used so
  each project can install what it needs without affecting others.
- **Anaconda** is a commercial distribution of Python+Conda+many
  packages that all work together.  It used to be freely usable for
  research, but since ~2023-2024 it's more limited.  Thus, we don't
  recommend it (even though it has a nice graphical user interface).
- **conda-forge** is another channel of distributing packages that
  is maintained by the community, and thus can be used by anyone.
  (Anaconda's parent company also hosts conda-forge packages)

> - **Miniforge** is a distribution of conda pre-configured for
>   conda-forge.  It operates via the command line.

Reviewer: ```suggestion
- **Miniforge** is the installer recommended by the conda-forge community. It operates via the command line.
```
From https://conda-forge.org/download/, where it states

> Miniforge is the preferred conda-forge installer

- **Miniconda** is a distribution of conda pre-configured to use
  the Anaconda channels.
:::

(miniforge)=

## Installing Miniforge

Download the latest release from the [Miniforge download
overview](https://conda-forge.org/download/).  This installs the base, and from
here other packages can be installed.

> 

Reviewer: This download page (which I did not know) is much easier to understand than the README.md at https://github.com/conda-forge/miniforge. 

Also, Chrome does allow the download from here.

Still, on Windows, Microsoft "Defender" "Smart" screen opposes  the installation, but it still works via powershell.

Author: Yeah I found it today for the first time :-)
